python/couch-app.py

```diff
@@ -67,10 +67,10 @@ def index():
     return app.send_static_file('index.html')
 
 # Serve up static resources, JavaScript, HTML, CSS
-@app.route('/<path:path>')
-def js_proxy(path):
+#@app.route('/<path:path>')
```

Comment:
oh now I get it, this is to serve the js/css contained in the front-end folder once I move its content here.